templates/kubernetes-cluster-with-new-vpc.template

```diff
@@ -451,7 +451,7 @@ Resources:
             curl -s -o $BASTION_BOOTSTRAP_FILE $BASTION_BOOTSTRAP
             chmod +x $BASTION_BOOTSTRAP_FILE
 
-            ./$BASTION_BOOTSTRAP_FILE --banner https://${QSS3BucketName}/${QSS3KeyPrefix}/scripts/banner_message.txt --enable true
+            ./$BASTION_BOOTSTRAP_FILE --banner https://s3.amazonaws.com/${QSS3BucketName}/${QSS3KeyPrefix}/scripts/banner_message.txt --enable true
```

Comment:
could you make this follow the convention in the rest of the repo?

`https://${QSS3BucketName}.s3.amazonaws.com/...`